targets/TARGET_STM/stm_spi_api.c

```diff
@@ -289,10 +366,21 @@ static void _spi_init_direct(spi_t *obj, const spi_pinmap_t *pinmap)
     handle->Init.FirstBit          = SPI_FIRSTBIT_MSB;
     handle->Init.TIMode            = SPI_TIMODE_DISABLE;
 
-#if TARGET_STM32H7
-    handle->Init.MasterKeepIOState = SPI_MASTER_KEEP_IO_STATE_ENABLE;
+#if defined (SPI_IP_VERSION_V2)
+    handle->Init.NSSPolarity = SPI_NSS_POLARITY_LOW;
+    handle->Init.MasterKeepIOState = SPI_MASTER_KEEP_IO_STATE_DISABLE;
```

Comment:
is the change from ENABLE to DISABLE expected ?

Reply:
I set SPI_MASTER_KEEP_IO_STATE_ENABLE back, as in Cube example, there is a often this comment:
```
SpiHandle.Init.MasterKeepIOState = SPI_MASTER_KEEP_IO_STATE_ENABLE;  /* Recommended setting to avoid glitches */
```